### ISSUE TITLE
Intellisense ought to show LongPtr as LongPtr

### ISSUE THREAD

**Issue by bclothier** [WaynePhillipsEA/twinbasic#693]
![image](https://user-images.githubusercontent.com/2367644/150909063-0c8e8d9f-b80a-422b-bbd3-dc93a878d6d9.png)

I get that it's showing the resolved data type (e.g. `Long` for `win32` build, `LongLong` for `win64` build). However, when authoring software, I think it's usually more helpful to see that it's actually a `LongPtr`, rather than the resolved `Long` or `LongLong`.

**Comment by WaynePhillipsEA** [comment]
I agree.   Actually, it would probably be useful to show both, but we don't want to make the signature difficult to read.
